.travis.yml

```diff
@@ -6,15 +6,18 @@ notifications:
     on_failure: change
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+  - 7.3
+  - 7.4
+
+services:
+  - mysql
```

Comment:
These are good fixes, but they should be moved into their own PR please.

Reply:
moved to fix/travis per your contribution!